code/modules/mob/living/carbon/alien/humanoid/caste/sentinel.dm

```diff
@@ -45,8 +45,10 @@
 	if(name == "alien sentinel")
 		name = text("alien sentinel ([rand(1, 1000)])")
 	real_name = name
-	verbs.Add(/mob/living/carbon/alien/humanoid/proc/corrosive_acid,/mob/living/carbon/alien/humanoid/proc/neurotoxin)
-	..()
+	internal_organs += new /obj/item/organ/internal/xenos/plasmavessel

```

Comment:
Ah, I think I might be misunderstanding how the code is structured, then - instead of the `internal_organs` list being a list of the organs present within the mob, it's a list of organs possible to insert into it?